src/backend/catalog/aclchk.c

```diff
@@ -5019,18 +5035,25 @@ RemoveRoleFromInitPriv(Oid roleid, Oid classid, Oid objid, int32 objsubid)
 			objid == get_database_oid(babelfish_db_name, true) &&
 			is_member_of_role(GetUserId(), sysadminOid = (*bbf_get_sysadmin_oid_hook)()))
 		{
-			grantorId = sysadminOid;
+
+			temp_acl = merge_acl_with_grant(old_acl,
+									   false,	/* is_grant */
+									   false,	/* grant_option */
+									   DROP_RESTRICT,
+									   list_make1_oid(roleid),
+									   ACLITEM_ALL_PRIV_BITS,
+									   sysadminOid,
+									   ownerId);
```

Comment:
This is a temporary fix. This will be reworked in the same cycle as a separate issue.